### PR TITLE
Use dismisser instead of cookies lib for onboarding tips

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: components/tips/view.coffee [hunk 1]
```diff
@@ -8,7 +8,10 @@ module.exports = class TipView extends Backbone.View
     'click': 'onClick'
     'click .js-close' : 'close'
 
-  initialize: ({ @user }) -> #
+  initialize: ({ @user }) ->
+    @dismisser = new Dismisser
+      key: @model.id
```
Comment (dzucconi) [inline components/tips/view.coffee]: What's model id evaluate to?

Comment (broskoski) [reply components/tips/view.coffee]: Something like so: https://github.com/aredotna/ervell/blob/master/apps/user/tips.coffee#L3

Comment (dzucconi) [reply components/tips/view.coffee]: ah cool. got it

File: components/tips/view.coffee [hunk 2]
```diff
@@ -52,6 +55,9 @@ module.exports = class TipView extends Backbone.View
         @remove()
 
   remove: ->
-    Cookies.set @model.id, true
+    @dismisser.dismiss()
```
Comment (dzucconi) [inline components/tips/view.coffee]: So wait where is the code that prevents the tips from loading

Comment (broskoski) [reply components/tips/view.coffee]: https://github.com/aredotna/ervell/blob/master/apps/user/tips.coffee#L3

Comment (broskoski) [reply components/tips/view.coffee]: https://github.com/broskoski/ervell/blob/77cfbd74d8fa5b9523876942ef8f1a8ae20d3047/components/tips/helpers.coffee#L4

Comment (broskoski) [reply components/tips/view.coffee]: Yeah, its a little weird. Tips get set on the route level (bc different tips for feed and for profile), so this adds all the tips that the route is trying to set, and rejecting the ones that already have cookies.

Comment (dzucconi) [reply components/tips/view.coffee]: It is odd. If it works it works. It'd probably be clearer to pass the dismissers into the templates or something, then use `if tip[id].dismisser.dismissed()`.

If this works for now that's fine as is.

Comment (broskoski) [reply components/tips/view.coffee]: Yeah that would be clearer, personally I think we should replace these entirely with a different approach.

Comment (dzucconi) [reply components/tips/view.coffee]: Yeah we will. You can access local data like cookies through an Apollo datalink, setting up a schema for it and so forth. So we'll go that route when we switch these pages over.

**Pull request merged**